dev-db/etcd/files/etcd.service

```diff
@@ -6,6 +6,8 @@ User=etcd
 PermissionsStartOnly=true
 Environment=ETCD_DATA_DIR=/var/lib/etcd
 Environment=ETCD_NAME=%m
-ExecStart=/usr/bin/etcd 
+ExecStart=/usr/bin/etcd
 Restart=always
 RestartSec=10s
+LimitNOFILE=1048576
```

Comment:
This value could very easily be higher than the limit for the entire system. Does it need to be that high?